src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -397,6 +618,79 @@ function isCtrlKey(e) {
 .sf-dump-str-expand .sf-dump-str-expand {
     display: none;
 }
+.sf-dump-public.sf-dump-highlight,
+.sf-dump-protected.sf-dump-highlight,
+.sf-dump-private.sf-dump-highlight,
+.sf-dump-str.sf-dump-highlight,
+.sf-dump-key.sf-dump-highlight {
+    background: rgba(111, 172, 204, 0.3);
+    border: 1px solid #7DA0B1;
+    border-radius: 3px;
+}
+.sf-dump-public.sf-dump-highlight-active,
+.sf-dump-protected.sf-dump-highlight-active,
+.sf-dump-private.sf-dump-highlight-active,
+.sf-dump-str.sf-dump-highlight-active,
+.sf-dump-key.sf-dump-highlight-active {
+    background: rgba(253, 175, 0, 0.4);
+    border: 1px solid #ffa500;
+    border-radius: 3px;
+}
+.sf-dump-search-hidden {
+    display: none;
+}
+.sf-dump-search-wrapper {
+    float: right;
+    font-size: 0;
+    min-width: 250px;
+    text-align: right;
+}
+.sf-dump-search-wrapper > * {
+    vertical-align: top;
+    box-sizing: border-box;
+    height: 21px;
+    font-weight: normal;
+    border-radius: 0;
+    background: #FFF;
+    color: #757575;
+    border: 1px solid #BBB;
+}
+.sf-dump-search-wrapper > input.sf-dump-search-input {
+    padding: 3px;
+    height: 21px;
+    font-size: 12px;
+    border-right: none;
+    width: 140px;
+    border-top-left-radius: 3px;
+    border-bottom-left-radius: 3px;
+    color: #000;
+}
+.sf-dump-search-wrapper > .sf-dump-search-input-next,
```

Comment:
Could be, but more specific selectors ensure it has priority, for instance over some more global styles defined and used for the WDT.